### PR TITLE
fix: correct issue with handling float32

### PR DESCRIPTION
**Pull request by irshadmb**
Error building Component Pinecone: Invalid type for variable '0'. Required value type is float and passed type was float64 at ['values'][0]

Fixed the issue by forcing float32

### CODE REVIEW

**Comment by ogabrielluiz** [comment]
Hey @irshadmb 

Could you update your code to create the class outside the Component, please? This is supported after #4316

**Comment by irshadmb** [comment]
@ogabrielluiz sure, will go through reference code and will do the changes

**Comment by irshadmb** [comment]
@ogabrielluiz updated and merged the code. Can you please review

File: src/backend/base/langflow/components/vectorstores/pinecone.py
Comment (irshadmb) [inline src/backend/base/langflow/components/vectorstores/pinecone.py:1]: @ogabrielluiz  - Its giving error: Error building Component: Float32Embeddings.__init__() got an unexpected keyword argument '_code'
Traceback (most recent call last):
  File "/mnt/d/github/langflow/src/backend/base/langflow/custom/utils.py", line 395, in build_custom_component_template
    has_template_config = hasattr(custom_component, "template_config")
  File "/mnt/d/github/langflow/src/backend/base/langflow/custom/custom_component/custom_component.py", line 408, in template_config
    self._template_config = self.build_template_config()
  File "/mnt/d/github/langflow/src/backend/base/langflow/custom/custom_component/base_component.py", line 97, in build_template_config
    component_instance = cc_class(_code=self._code)
TypeError: Float32Embeddings.__init__() got an unexpected keyword argument '_code'

**Comment by irshadmb** [comment]
@ogabrielluiz - I have made some re-arranging of code. Then it has started working. I have done the commit. Also code is tested now: 

image

**Review by ogabrielluiz — APPROVED** [review]

**Pull request merged**